### PR TITLE
Fix CVV Code Generation

### PR DESCRIPTION
**Pull request by kevinastone**
Mastercard and American Express were flipped.  AmEx has 4 digit CVV codes and Mastercard has 3 like most of the other cards.

http://en.wikipedia.org/wiki/Card_security_code

### CODE REVIEW

File: faker/providers/credit_card/__init__.py [hunk 1]
```diff
@@ -118,7 +118,7 @@ def credit_card_full(self, card_type=None, validate=False, max_check=10):
 
     @classmethod
     def credit_card_security_code(cls, card_type=None):
-        return cls.random_number(cls._credit_card_type(card_type).security_code_length)
+        return ''.join(str(cls.random_digit()) for _ in xrange(cls._credit_card_type(card_type).security_code_length))
```
Comment (fcurella) [inline faker/providers/credit_card/__init__.py]: can you use `range` instead of `xrange` for compatibility with Python 3.x?

**Comment by kevinastone** [comment]
@fcurella good spot.  fixed.

**Comment by fcurella** [comment]
@kevinastone I don't think we need to worry about providing a compatibility layer. We're `range`-ing for `cls._credit_card_type(card_type).security_code_length`, which is either `3` or `4`. Compared to `range, `xrange` won't improve performance that much on Python 2.

I'd rather keep it simple and just use `range` across all Python versions.

**Comment by kevinastone** [comment]
@fcurella fair enough, pushed/reverted to `range`.

**Comment by fcurella** [comment]
Thanks @kevinastone , really appreciated it! :)

**Pull request merged**